src/editors/containers/VideoUploadEditor/hooks.js

```diff
@@ -7,15 +7,15 @@ export const {
   navigateTo,
 } = appHooks;
 
-export const postUploadRedirect = (storeState) => {
+export const postUploadRedirect = (storeState, uploadType) => {
```

Comment:
Can you add a default value here? e.g. `uploadType = selectedVideoUrl`?

since `${undefined}` evaluates to "undefined" and we get a 404 if someone doesn't add the parameter.

Unless you want a 404 produced in this case so we know something's up.